Islandora-Devops/claw-playbook
# Upgrade 5.4 symfony and separate EntityMapper

**GitHub Issue**: https://github.com/Islandora/documentation/issues/2220

Relies on and tests:
* https://github.com/Islandora/chullo/pull/99
* https://github.com/Islandora/Crayfish-Commons/pull/63
* https://github.com/Islandora/Crayfish/pull/174
* https://github.com/Islandora/islandora/pull/936

# What does this Pull Request do?

Builds a box to test the above PRs

# What's new?

Updates templated files to remove unnecessary parts and work with Symfony 5.4 

* Does this change require documentation to be updated? no
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# How should this be tested?

The base box needs to be rebuilt and then you need to build the vagrant box using the `starter_dev` profile to pull in the correct "starter site"
i.e.
```
> vagrant box remove islandora_base
> ISLANDORA_BUILD_BASE=true vagrant up 
> vagrant package --output islandora_base
> ISLANDORA_BUILD_BASE=true vagrant destroy
> ISLANDORA_INSTALL_PROFILE=starter_dev vagrant up
```

Then you should have a normal working site, if you `vagrant ssh` in you should see the changes from the various PRs listed above in the code.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora-Devops/committers
